developer/contributing.md

```diff
@@ -13,27 +13,52 @@ cd reaction
 reaction
 ```
 
-## Step 2: Finding an issue
+## Step 2: Find or open an issue
 
 Explore the interface and the code to give you a good overview of the product and a sense for what's already built. Keep an eye out for bugs and interface issues, as well as features you'd like to see created.
 
-Check our our [`Good First Issue`](https://github.com/reactioncommerce/reaction/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) and [`Help Wanted`](https://github.com/reactioncommerce/reaction/issues?q=label%3A%22help+wanted%22) tags for good issues to tackle. If you find something you want to work on, let us know right there in the comments. If you are interested in a specific aspect of the project but aren't sure where to begin, feel free to ask.
+There are two ways to go about contributing to Reaction: work on an issue or bug that is already created and vetted by the team, or propose a new feature in our [Reaction Feature Requests](https://github.com/reactioncommerce/reaction-feature-requests) repository.
 
-Start small and open up a dialogue with us. This will help to get your contributions accepted easily.
+### Find an issue and claim it
 
-## Step 3: The pull request and review process
+1. Explore the [Help Wanted](https://github.com/reactioncommerce/reaction/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22) or [Good First Issue](https://github.com/reactioncommerce/reaction/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) issues on our GitHub repo.
+2. If you find something you want to work on, let us know right there in the comments.
```

Comment:
@zenweasel Should I add a step 3 here to officially request to become a Contributor and to wait until your GitHub repo permission status is changed?